corehq/apps/es/users.py

```diff
@@ -150,7 +145,3 @@ def user_ids_at_locations_and_descendants(location_ids):
 
 def user_ids_at_locations(location_ids):
     return UserES().users_at_locations(location_ids).exclude_source().run().hits
-
```

Comment:
Hey, thanks for pointing this out. I just checked on it, and when I use `user_ids_at_locations()` in the scope filter, the locations don't come straight from a user's assigned locations, I use `SQLLocation.active_objects.accessible_location_ids(domain, couch_user))` to get the locations, which is the same thing that `UserES.users_at_accessible_locations()` does, so I think this should be safe.

I'll add back the `user_ids_at_accessible_locations()` function here to avoid the issues with the other branches on staging.